src/test/java/com/google/sps/servlets/UserDataServletTest.java

```diff
@@ -108,7 +109,8 @@ public void testGetMethodWithInvalidId() throws Exception {
         USER_FOUND_PROPERTY, false);
 
     // Check that the user wasn't found in datastore
-    assertThat(gson.toJson(expected)).isEqualTo(stringWriter.toString());
+    Map<String, Object> actual = gson.fromJson(stringWriter.toString(), Map.class);
+    assertThat(expected).containsEntry(USER_FOUND_PROPERTY, actual.get(USER_FOUND_PROPERTY));
```

Comment:
General assertion ordering is actual -> expected.

Here, as I've looked a tad more into Truth, we can actually do this (no need for the expected map):

```
assertThat(actual).containsEntry(USER_FOUND_PROPERTY, false);
```